pkg/registry/cachesize/cachesize.go

```diff
@@ -105,6 +105,10 @@ func SetWatchCacheSizes(cacheSizes []string) {
 			glog.Errorf("invalid size of watch cache capabilities: %s", c)
 			continue
 		}
+		if size < 0 {
+			glog.Errorf("watch cache size cannot be negative")
```

Comment:
Changed to `glog.Errorf("watch cache size cannot be negative: %s", c)`, PTAL. Thanks!